app/controllers/v1/registrations_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module V1
+  class RegistrationsController < ApiController
+    def create
+      user = User.new(params.permit(:email, :password))
+      # render json: user, status: :created if user.save
+      if user.save
+        respond_with(status: 201, entity: user, serializer: RigistrationSerializer)
```

Comment:
```ruby
Api::V1::Users::Registrations::Serializer::Create
```

Reply:
Api::V1::Users::Registrations::Serializer::CreateSerializer. Without word Serializer in class name doesn't work(